src/components/FeedbackButton/FeedbackButton.js

```diff
@@ -0,0 +1,160 @@
+
+type Props = {
+  size: number,
+  color: string,
+  hoverColor: string,
+};
+
+type State = {
+  isHovered: boolean,
+};
+
+class FeedbackButton extends PureComponent<Props, State> {
+  static defaultProps = {
+    size: 28,
+    color: COLORS.gray[600],
+    hoverColor: COLORS.purple[500],
+  };
+
+  state = {
+    isHovered: false,
+  };
+
+  handleMouseEnter = () => {
+    this.setState({
+      isHovered: true,
+    });
+  };
+  handleMouseLeave = () => {
+    this.setState({
+      isHovered: false,
+    });
+  };
+
+  render() {
+    const { color, hoverColor, size, ...delegatedProps } = this.props;
+    const { isHovered } = this.state;
+
+    return (
+      <Spring
+        native
+        config={{
+          tension: 190,
+          friction: 40,
+        }}
+        to={{ hovered: isHovered ? 1.0 : 0 }}
+      >
+        {interpolated => (
+          <Wrapper
+            native
+            {...delegatedProps}
+            onClick={() => shell.openExternal(IN_APP_FEEDBACK_URL)}
+            onMouseLeave={this.handleMouseLeave}
+          >
+            <SliderWrapper style={{ cursor: isHovered ? 'pointer' : 'auto' }}>
+              <SliderBox
+                hovered={interpolated.hovered}
+                style={{
+                  transform: interpolate(
+                    [interpolated.hovered],
+                    scale => `scale(${Math.max(scale * 180, 1)}, 1)`
+                  ),
+                  opacity: interpolate(
+                    [interpolated.hovered],
+                    opacity => opacity
+                  ),
+                }}
+              />
+              <Text
+                style={{
+                  transform: interpolate(
+                    [interpolated.hovered],
+                    pos => `translateX(-${Math.max(pos * 110, 5)}px)`
+                  ),
+                }}
+              >
+                Feedback
+              </Text>
+            </SliderWrapper>
+            <Icon
+              size={size}
+              icon={messageSquare}
+              style={{
+                transform: interpolate(
+                  [interpolated.hovered],
+                  scale => `scale(${scale / 5 + 1}, ${scale / 5 + 1})`
+                ),
+              }}
+              color={!!isHovered ? hoverColor : color}
+              onMouseEnter={this.handleMouseEnter}
+            />
+          </Wrapper>
+        )}
+      </Spring>
+    );
+  }
+}
+
+const Wrapper = styled.div`
+  position: fixed;
+  right: 15px;
+  bottom: 15px;
+  width: 50px;
+  height: 50px;
+  text-align: center;
+  z-index: +1;
+`;
+
+const SliderBox = animated(styled.div`
+  position: absolute;
+  right: -25px;
+  width: 1px; /* small width so it hides behind wrapper - not 0 because we need something to scale*/
+  height: 50px;
+  /*
+  // would be nice to have a border-radius but scale is modifying it
+  // so it's looking weird - I think it's OK to just have the round clipping from SliderWrapper overflow

```

Comment:
The clipping kinda bugs me :/ It feels like a visual glitch, since it's not a gradual rounding animation, it's just being occluded by an invisible wrapper.